packages/editor/src/lib/editor/Editor.ts

```diff
@@ -1915,6 +1915,43 @@ export class Editor extends EventEmitter<TLEventMap> {
 		return this
 	}
 
+	/**
+	 * Calculate and update the currently hovered shape based on the current page point.
+	 *
+	 * @public
+	 */
+	updateHoveredId() {
```

Comment:
Felt strange that this was in the `tldraw` package since it needs so much from the editor? Might be better if it lives here?